<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-FMC8G94" failures="4" tests="4" name="Default test(failed)" time="38.931" errors="0" timestamp="2023-01-06T20:51:31 IST">
  <testcase classname="TestCases.FaildTest" name="Test1" time="0.123">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at TestCases.FaildTest.Test1(FaildTest.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- Test1 -->
  <testcase classname="TestCases.FaildTest" name="Test5" time="0.073">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at TestCases.FaildTest.Test5(FaildTest.java:58)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- Test5 -->
  <testcase classname="TestCases.FaildTest" name="Test3" time="0.076">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at TestCases.FaildTest.Test3(FaildTest.java:44)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- Test3 -->
  <testcase classname="TestCases.FaildTest" name="Test2" time="0.081">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at TestCases.FaildTest.Test2(FaildTest.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- Test2 -->
  <testcase classname="TestCases.FaildTest" name="Test4" time="0.0">
    <ignored/>
  </testcase> <!-- Test4 -->
</testsuite> <!-- Default test(failed) -->
